### PR TITLE
Feat: #944 Add a flag to not translatable strings

### LINKED ISSUE #944: Feature Request: Add translatable=false flag to make translations easier to do
**Summary:**

There are some strings which shouldn't be translated.

**Steps to reproduce:**

app>src>main>res>values>strings.xml

**Expected behavior:**

Add translatable="false" flag to non translatable strings.

**Observed behavior:**

There are some strings which shouldn't be translated, but these don't have the flag.

### PR DESCRIPTION
**Pull request by letelete**
Fixes #944 

Please Add Screenshots If there are any UI changes.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -412,26 +412,26 @@
...
     <!--Format Strings-->
     <string name="hello_client">Hello, %1$s</string>
-    <string name="double_amount">%1$.2f</string>
-    <string name="double_and_string">%1$.2f %2$s</string>
-    <string name="string_and_string">%1$s %2$s</string>
-    <string name="string_and_double">%1$s %2$.2f</string>
-    <string name="string_and_int">%1$s: %2$d</string>
+    <string name="double_amount" translatable="false">%1$.2f</string>
+    <string name="double_and_string" translatable="false">%1$.2f %2$s</string>
+    <string name="string_and_string" translatable="false">%1$s %2$s</string>
```
Comment (m-sameer) [inline app/src/main/res/values/strings.xml:417]: For the right to left languages, the order of the two parameters has to be flipped. On the right side will be the first parameter and on the left side will be the second parameter. So I think you should keep it translatable.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -412,26 +412,26 @@
...
+    <string name="string_and_double" translatable="false">%1$s %2$.2f</string>
+    <string name="string_and_int" translatable="false">%1$s: %2$d</string>
     <string name="invalid_amount">Invalid Amount</string>
     <string name="exit_message">Press back again to exit</string>
 
     <string name="package_name" translatable="false">package</string>
     <string name="permission_denied_camera">Permission Denied to use Camera</string>
-    <string name="one">1</string>
```
Comment (m-sameer) [inline app/src/main/res/values/strings.xml:425]: For some language different numerals are used for example in Hindi, Urdu, Arabic, etc. So this must also be kept as translatable.

**Comment by letelete** [comment]
I'm closing this pr anyway since almost all translations are done.

**Comment by m-sameer** [comment]
Also, I have found a string `pref_configuration` which is used as a key in preferences, therefore it should not be translated to any other language.

**Comment by m-sameer** [comment]
I don't think you should close this PR, we only have translations in about 15 languages. Many of them can be added in future.

**Comment by m-sameer** [comment]
@letelete If you're not working on the issue may I work on it?

**Comment by letelete** [comment]
Sure

**Pull request closed without merging**